### ISSUE TITLE
Fix transactional support in Wildfly Example

### ISSUE THREAD

**Issue by zambrovski** [holisticon/axon-cdi#9]
Currently, Wildfly example works only, if no transaction suport is activated.

Fix it.

**Comment by jangalinski** [comment]
with current master, running `./mvnw -f example/wildfly` fails with

```
Caused by: javax.ejb.EJBException: org.axonframework.commandhandling.NoHandlerForCommandException: No handler was subscribed to command [org.axonframework.extension.example.common.core.api.CreateAccountCommand]
```

is this what you mean with transaction support or is this st else?

**Comment by zambrovski** [comment]
No, I just brkoe it again, by extracting common classes. This is a different issue.

**Comment by alexmacavei** [comment]
I'm getting this with TxAwareJPAEventStorageEngine:

java.lang.IllegalArgumentException: Unknown entity: org.axonframework.eventsourcing.eventstore.jpa.DomainEventEntry

Are these related I wonder?

**Comment by zambrovski** [comment]
This issue is solved in #13
